tests/stackage_zlib_runpath/BUILD.bazel

```diff
@@ -13,20 +13,21 @@ dynamic_libraries(
     srcs = ["@zlib.dev//:zlib"],
     filter = "libz",
     solib_names = "libz_soname",
-    tags = ["requires_zlib"],
+    # Only the Nix provided zlib is dynamically linked.
+    tags = ["requires_nix"],
```

Comment:
These tests don't require any `zlib`, but specifically the one provided by nixpkgs. So, the tag is named `requires_nix` instead of `requires_zlib`.